src/configurations/destinations/braze/ui-config.json

```
@@ -12,66 +12,326 @@
                 "note": "Update your connection settings here",
                 "icon": "settings",
                 "fields": [
+                  {
+                    "type": "checkbox",
+                    "label": "Enable Platform-Specific API Keys",
+                    "configKey": "usePlatformSpecificApiKeys",
+                    "default": true,
```

We need to change the default value to `false`. Since this is wrapped in a feature flag, the user won’t have an option to disable it, and because the “Default API Key” is only visible when this is set to `false`, that field will remain hidden if it isn’t.

```suggestion
                    "default": false,
```

We can set it back to true, when we remove the feature flag.